app/models/comment.rb

```diff
@@ -29,7 +29,8 @@ class Comment < ActiveRecord::Base
   # I *do not* have any idea why Time.now wasn't working (I assume it was a time-zone thing), but this works:
   scope :visible, -> { where('visible_at <= ?', 0.seconds.from_now) }
   scope :undeleted, -> { where(deleted: false) }
-
+  scope :comments_of_today, ->(current_user) { where(user_id: current_user.id,
```

Comment:
Use the `lambda` method for multi-line lambdas.<br>Block body expression is on the same line as the block start.